### ISSUE TITLE
Microsoft WCAG 2.2 CR2 Feedback: 2.4.11 Focus Appearance

### ISSUE THREAD

**Issue by dbjorge** [w3c/wcag#3037]
I met with Microsoft CELA's Accessibility Technology group over the last two weeks to gather collective feedback on CR2 changes. Their feedback is aligned with the Microsoft's Accessibility Insights team's feedback from the perspective of a tool developer.

The parts we had specific new feedback on were the removal of 4.1.1 Parsing (separate issue #3034) and whether we supported keeping vs removing the at-risk 2.4.11 (this issue).

On 2.4.11 Focus Appearance:

> * The draft/language of 2.4.11 Focus Appearance is very difficult to implement. The current text with 2.4.11 Focus Appearance is making it harder and should be providing clarity. With this said, because of the dynamic nature of the web, web pages are becoming denser and therefore guidance is needed to help with this. WCAG guidance needs to be written in a much easier to digest fashion. 
> * 2.4.11 needs to be postponed until cleaner, easier to read and understand text is written. 

We are very supportive of the intent of the SC, and there was substantial internal debate around whether the merits of the intended outcome of the SC outweighed the understandability and implementation difficulty concerns.

However, in the end, we felt that the SC as currently written is too complicated to understand and too difficult to implement for us to feel comfortable supporting it as a recommendation in its current state. We support dropping the at-risk SC from WCAG 2.2 unless/until it can be substantially simplified.

Some of our experiences that we felt justified this conclusion were:

* We attempted to write up step-by-step instructions for manually validating an element in simple, clear language, in the same style that we currently use to write similar instructions for other WCAG criteria as part of Accessibility Insights for Web's Assessment. Our most successful draft of this was approximately three times as long as the currently-most-complex requirement (24.2 Contrast > State changes). This was despite the draft being incomplete; it would have needed to become more complex to handle certain edge cases (for example, the special case in the "minimum bounding box" definition for line-wrapped links)
* We gathered subjective feedback from of employees in different roles (engineers, PMs, legal, compliance, accessibility domain experts) and with different levels of accessibility expertise. Folks across all roles and expertise levels found it to be subjectively more confusing than other comparable SCs.
* We spent approximately 2 engineer-weeks in Accessibility Insights researching and prototyping how we might automate a test for the requirement (in whole or part). We didn't come up with anything very satisfactory. We concluded that in practice it's probably infeasibly complicated to try to approach this by comparing browser-reported stylings of different components, and that to approach it reliably you'd need to do image-diffs of each component of a page in a focused vs unfocused state as-rendered by the browser. But we think an image-diff-based solution would *also* be unreliable for indicators that are heavy anti-aliased. Additionally, automating a test for this is complicated by the lack of browser support for tools like ours to simulate focus states without user interaction.
* I completed the exercises in [Testing Focus Appearance #2901](https://github.com/w3c/wcag/issues/2901). In general, I felt that there was an uncomfortable number of real-world cases from this exercise that required subjective tester judgement and/or which required onerous pixel-by-pixel analysis to assess accurately. Full results and notes were shared privately with the AG chairs per the instructions in that issue.

**Comment by alastc** [comment]
This will be closed by the PR for CR, #3123
